package.json

```diff
@@ -108,7 +108,7 @@
     "redux-thunk": "^2.3.0",
     "request": "2.87.0",
     "reselect": "^3.0.1",
-    "selfkey-ui": "https://github.com/SelfKeyFoundation/selfkey-ui.git",
+    "selfkey-ui": "https://github.com/SelfKeyFoundation/selfkey-ui.git#fix/no-gas",
```

Comment:
Prefer using commit hashes. This will fail if that branch gets merged and deleted.

Reply:
@dfcowell yes, I usually update after selfkey-ui was merged. The hash can change after merge no?
Will update now